### DISCUSSION TITLE
Welcome to Scientific-Calculator-Project Discussions!

### DISCUSSION THREAD

**Discussion by wasif-h** [wasif-h/Scientific-Calculator-Project#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wasif-h** [comment]
@saidulislam2003 

Now what should we do next ?
What functionality should we add ?

**Reply by saidulislam2003** [reply]
@wasif-h 
Now we can add math library then for every part of math functionalities we can make another cpp file like for trigonometry, we will create trigonometry.cpp and we will just work on trigonometrical terms like sin cos tan etc.

**Reply by wasif-h** [reply]
Well explained. Thank you

**Reply by saidulislam2003** [reply]
@wasif-h 
check pull request

**Reply by wasif-h** [reply]
@saidulislam2003 
Yes , I merged them.
Great Work Saidul.

**Note : Please write an informative message at commit**

**Reply by saidulislam2003** [reply]
Ok. I'll try it next time. Thanks for your feedback.
